server/fishtest/templates/machines_table.mak

```diff
@@ -16,6 +16,17 @@
   </thead>
   <tbody>
     % for machine in machines:
+        <%
+          if 'python_version' in machine:
+            gcc_version = ".".join([str(m) for m in machine['gcc_version']])
+            python_version = ".".join([str(m) for m in machine['python_version']])
+            version = machine['version']
+          # This code may be deleted when all connected workers have upgraded.
+          else:
+            gcc_version = machine['gcc_version']
```

Comment:
Reading the code on the fly, if I'm not wrong this could be simplified after the workers bump to the new version.

Reply:
Yes this can go away after the workers have upgraded. The other gluing code in `tests_view.mak` can go away after converting the db. 

I am currently working on this. To start I have written (for myself) a basic mongo shell modeled after a standard unix shell. I find the standard mongo shell to be basically unusable.